lm_eval/tasks/score/mmlu_pro/prompt_templates.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
.json files should also have a license header

Reply:
We cannot add comments in a json file